request.go

```diff
@@ -42,30 +42,30 @@ type Requests struct {
 	StatusCode int          `json:"statuscode,omitempty"`
 }
 
+type RequestMetadata struct {
+	CreatedDate   time.Time     `json:"createdDate"`
+	CreatedBy     string        `json:"createdBy"`
+	Etag          string        `json:"etag"`
+	RequestStatus RequestStatus `json:"requestStatus"`
+}
+
+type RequestProperties struct {
+	Method  string      `json:"method"`
+	Headers interface{} `json:"headers"`
```

Comment:
now that you changed `Body` to be a string and this is a breaking change anyway, you could probably also make `Headers` `*http.Header` as it is everywhere else

Reply:
The headers in this struct is a nested dictionary that is part of the properties field. The others are real http.Headers. So it should be left like it is.

Reply:
ok